src/objects/media/media.scss

```diff
@@ -50,3 +52,49 @@
 .o-media__content {
   flex: 1;
 }
+
+/**
+ * We use CSS grid when supported by the browser.
+ */
+
+@supports (display: grid) {
```

Comment:
Since IE11 is the only browser that doesn't support Grid these days, I wonder if having a fallback is still necessary?